### ISSUE TITLE
Styles for other languages?

### ISSUE THREAD

**Issue by theryan722** [jacobslusser/ScintillaNET#38]
Hello, what you have done is incredible here, so thank you.

I see that there are styles for a few languages, what if I want to set a style for a language such as Ada? Do I have to modify and recompile ScintillaNET myself to add Ada to ScintillaNET.Style?

While the CPP style works for several languages, my editor has to support a lot of languages, and not all have "//" as the comment delineation.

Thanks.

**Comment by theryan722** [comment]
Okay, so I modified ScintillaNET and opened up the lexers and created the styles for each language I needed (added the constants to nativemethods, then the class in style, etc.). However everytime I try to set the colors for the styles via code, the application suddenly says its not working and thats it, even if I am setting it for a language such as C++ which had the style stuff already set up for it. Now I am frustrated because I spent hours looking through the Scintilla documentation to get the proper style information and whatnot. Any ideas?
